apps/approval/appconfig.py

```diff
@@ -4,3 +4,7 @@
 class ApprovalConfig(AppConfig):
     name = 'apps.approval'
     verbose_name = 'Approval'
+
+    def ready(self):
+        super().ready()
+        import apps.approval.signals  # flake8: noqa
```

Comment:
This is to prevent flake8 to report this as an unused import.

> lines that contain a # noqa comment at the end will not issue warnings.

Source: https://pypi.python.org/pypi/flake8